src/main/java/org/terasology/climateConditions/BodyTemperatureSystem.java

```diff
@@ -55,6 +55,10 @@ public void postBegin() {
         // The world is the entity here which shall have a Body Temperature Update Periodic Action.
     }
 
+    /**
+     * Updates the Body Temperature System when the periodic action with actionId = BODY_TEMPERATURE_UPDATE_ACTION_ID is
```

Comment:
> when the periodic action with actionId = BODY_TEMPERATURE_UPDATE_ACTION_ID is triggered

This is somewhat of an implementation detail... 🤔 it would be nice to write something like "once a second" or so, but that obviously can change unnoticed leaving the documentation out of date...